app/screens/user_profile/user_info.tsx

```diff
@@ -22,41 +23,27 @@ type Props = {
     enableCustomAttributes?: boolean;
 }
 
-const emptyList: DisplayCustomAttribute[] = []; /** avoid re-renders **/
+const emptyList: CustomAttribute[] = []; /** avoid re-renders **/
 
 const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user, enableCustomAttributes}: Props) => {
     const customStatus = getUserCustomStatus(user);
     const serverUrl = useServerUrl();
-    const [customAttributes, setCustomAttributes] = useState<DisplayCustomAttribute[]>(emptyList);
+    const [customAttributes, setCustomAttributes] = useState<CustomAttribute[]>(emptyList);
+
     const lastRequest = useRef(0);
 
+    const sortAttributes = useCallback(sortCustomProfileAttributes, []);
+
     useEffect(() => {
         if (enableCustomAttributes) {
             const fetchData = async () => {
                 const reqTime = Date.now();
                 lastRequest.current = reqTime;
-                try {
-                    const client = NetworkManager.getClient(serverUrl);
-                    const [fields, attrValues] = await Promise.all([
-                        client.getCustomProfileAttributeFields(),
-                        client.getCustomProfileAttributeValues(user.id),
-                    ]);
-
-                    // ignore results if there was a newer request
-                    if (fields && fields.length > 0 && lastRequest.current === reqTime) {
-                        const attributes = fields.map((field) => {
-                            if (attrValues[field.id]) {
-                                return ({
-                                    id: field.id,
-                                    name: field.name,
-                                    value: attrValues[field.id] || '',
-                                } as DisplayCustomAttribute);
-                            }
-                            return {} as DisplayCustomAttribute; // this will be cleaned out in CustomAttributes along with the fixed attributes.
-                        });
-                        setCustomAttributes(attributes);
-                    }
-                } catch {
+                const {attributes, error} = await fetchCustomAttributes(serverUrl, user.id, true);
+                if (!error && lastRequest.current === reqTime) {
+                    const attributesList = Object.values(attributes).sort(sortAttributes);
```

Comment:
0/5 I don't see new tests regarding these changes (the attributes show sorted on render).